src/index.html

```diff
@@ -17,11 +17,381 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="style.scss"
```

Comment:
The link to the stylesheet should point to a .css file, not a .scss file. The browser won't understand SCSS syntax. You should compile your SCSS to CSS and link the generated CSS file here.